scinet/__init__.py

```diff
@@ -1,6 +1,8 @@
 '''
-Scinet package, contains Scinet class and Hyperparameters class
+Scinet package, contains Scinet class and Hyperparameters class and
+Visualization functions
 '''
 
 from .scinet import Scinet
 from .hyperparameters import Hyperparameters
+from .visualization import plot_loss, plot_latent
```

Comment:
Changed this to be more explicit, to avoid polluting the namespace too much.
Also remember that this puts things into the `scinet` namespace, so you can call things like `scinet.plot_loss` you don't need to be importing visualization itself.